apps/spark/src/test/java/com/linkedin/openhouse/catalog/e2e/SparkSchemaEvolutionTest.java

```diff
@@ -0,0 +1,86 @@
+package com.linkedin.openhouse.catalog.e2e;
+
+import com.linkedin.openhouse.jobs.spark.Operations;
+import com.linkedin.openhouse.tablestest.OpenHouseSparkITest;
+import java.util.Arrays;
+import java.util.List;
+import org.apache.iceberg.Schema;
+import org.apache.iceberg.Table;
+import org.apache.iceberg.spark.SparkSchemaUtil;
+import org.apache.iceberg.types.Types;
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.RowFactory;
+import org.apache.spark.sql.SparkSession;
+import org.apache.spark.sql.types.DataTypes;
+import org.apache.spark.sql.types.Metadata;
+import org.apache.spark.sql.types.StructField;
+import org.apache.spark.sql.types.StructType;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class SparkSchemaEvolutionTest extends OpenHouseSparkITest {
+
+  @Test
+  void testEndUserSchemaEvolution() throws Exception {
+    SparkSession spark = null;
+    try {
+      spark = getSparkSession();
+      spark.sql("CREATE TABLE openhouse.d1.t1 (name string, id bigint)");
```

Comment:
maybe you want add a test-case that starts with name, id, dt as the schema, and then wants to evolve to name, id, desc, dt?

Reply:
It's technically the same as the current case, where `schema` at line 39 is intentionally putting `dt` in the very beginning. Client needs to do things at Line 62 - 66 to make sure it presents the expected schema where `dt` will be placed at the end.

Reply:
Confused, the client wants the schema to be dt, id, name, but the test is written in a way that it becomes id, name, and dt?